aiosmtpd/docs/_exts/autoprogramm.py

```diff
@@ -0,0 +1,349 @@
+"""
+    autoprogramm
```

Comment:
This Sphinx 'private extension' extracts documentation from the `ArgumentParser` construct. See the rest of the docstring for changes / source / details / etc.